ajax/libs/d3-request/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "d3-request",
+  "filename": "d3-request.min.js",
+  "description": "A convenient alternative to XMLHttpRequest.",
+  "keywords": [
+    "d3",
+    "d3-module",
+    "request",
+    "ajax",
+    "XMLHttpRequest"
+  ],
+  "homepage": "https://d3js.org/d3-request/",
+  "license": "BSD-3-Clause",
+  "author": {
+    "name": "Mike Bostock",
+    "url": "https://bost.ocks.org/mike"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/d3/d3-request.git"
+  },
+  "npmName": "d3-request",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/*"
```

Comment:
@kyrylo I think we should exclude `d3-request.node.js`. Let's use `"**/!(*node*)"` instead. Thanks.